master/internal/experiment.go

```diff
@@ -633,6 +639,29 @@ func (e *experiment) setOrder(ctx *actor.Context, queuePosition float64) error {
 	return nil
 }
 
+func (e *experiment) setRP(ctx *actor.Context, msg job.SetRP) error {
+	if sproto.UseK8sRM(ctx.Self().System()) {
+		ctx.Respond(fmt.Errorf("kubernetes does not support setting resource pools"))
+	}
```

Comment:
break the execution flow?
also return the error and leave responding to the `Receive` method? if that's how we do it elsewhere.

Reply:
in this method sometimes we respond and sometimes return an error we'd probably want one.

Reply:
will fix